.github/workflows/release_pull_request.yml

```diff
@@ -30,6 +30,10 @@ jobs:
       devPrUrl: ${{steps.urls.outputs.dev}}
       releaseHash: ${{steps.urls.outputs.release}}
     steps:
+      - name: 'Define branch variables'
+        run: |
+          echo "MERGE_TO_DEV_BRANCH=release-${{inputs.version}}-to-develop" >> $GITHUB_ENV
+          echo "MERGE_TO_MAIN_BRANCH=merge-${{inputs.version}}-to-main" >> $GITHUB_ENV  
```

Comment:
Is there a reason why this is its own step and they are set as workflow env vars instead of just normal bash variables within each of the steps? 

Workflow vars are typically used when they need to be used across multiple steps. These branch variables are only used within their own step so you probably don't need to define them at the workflow env level. See the following example in [release_branch_issue.yml](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/4f29195968a96a371a5726b7b8402638b3fd1913/.github/workflows/release_branch_issue.yml#L70)